exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -105,6 +109,25 @@ public String getTime() {
       return format.format(time);
     }
 
+    public long getStartTime() {
+      return startTime;
+    }
+
+    public long getEndTime() {
+      return endTime;
+    }
+
+    public String getDuration() {
```

Comment:
How about we create wrapper class around `QueryProfile` (ex: `QueryProfileWrapper`) which will contain `getDuration` method along with needed `QueryProfile` methods. Logic for converting two long values (start and end) into human-readable format will be factored out into static method in `ProfileResources` (ex: `convertDurationIntoString(long start, long end)`). `QueryProfileWrapper` and `ProfileResources` will call this utility method from their `getDuration` methods.